docs/image-config/README.md

```diff
@@ -0,0 +1,142 @@
+# Configuring the K8ssandra images
+
+Often users find it neccesary to configure various properties of the images they are using, this may be to:
```

Comment:
Necessary

Reply:
That's embarrassing. Fixed. Good thing one of us can spell around here.